components/tikv_util/src/logger/mod.rs

```diff
@@ -165,6 +166,22 @@ pub fn get_string_by_level(lv: Level) -> &'static str {
     }
 }
 
+pub fn redirect_panic_log(panic_log_file: &str) {
+    if panic_log_file.is_empty() {
+        return;
+    }
+    let tikv_err_log_file = OpenOptions::new()
+        .create(true)
+        .write(true)
+        .append(true)
+        .open(panic_log_file)
+        .unwrap();
+
+    let decorator = slog_term::PlainDecorator::new(tikv_err_log_file);
+    let drainer = TikvFormat::new(decorator);
+    let _ = init_log(drainer, slog::Level::Error, false, false, vec![], 0);
```

Comment:
Does it mean all error logs are dupped to this file? Only panic information is wanted.

Reply:
only used in panic hook, so just panic log will redirect